test.py

```diff
@@ -4,8 +4,10 @@
 import unittest
 
 class MathTestCase(unittest.TestCase):
-    def verify(self, mkd_name, html_name, enable_dollar_delimiter=False):
-        config = {'enable_dollar_delimiter': enable_dollar_delimiter}
+    def verify(self, mkd_name, html_name, enable_dollar_delimiter=False,
+            render_to_span=False):
+        config = {'enable_dollar_delimiter': enable_dollar_delimiter,
```

Comment:
I'd rather not add a test. I thought it more logical to add an argument even if currently unused, in case some test for it were added later. But OK, I'll remove it.